content-resources/src/main/resources/flyway/oskari/V2_0_8__register_announcement_bundles.java

```diff
@@ -0,0 +1,20 @@
+package flyway.oskari;
+
+import java.sql.Connection;
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+
+import org.oskari.helpers.BundleHelper;
+import fi.nls.oskari.domain.map.view.Bundle;
+
+public class V1_0_0__register_announcement_bundles extends BaseJavaMigration {
```

Comment:
The class name should match the filename: 
```suggestion
public class V2_0_8__register_announcement_bundles extends BaseJavaMigration {
```

Reply:
Had some changes that wasn't pushed, including this fix